packit/api.py

```diff
@@ -181,7 +188,7 @@ def sync_release(
             self.dg.update_branch(dist_git_branch)
             self.dg.checkout_branch(dist_git_branch)
 
-            if create_pr:
+            if self._is_create_pr_allowed(create_pr=create_pr):
```

Comment:
why not `if self.package_config.create_pr or create_pr`?

Reply:
Yeah, it could be, but in my mind raises another use case.

Do we count, that CLI option overrides `packit.yaml` configuration?
If so, then `self.is_create_pr_allowed` can remain.
Otherwise, your proposal seems to be fine.

Reply:
yes, CLI has higher prio than config file

Reply:
okay, it makes sense then to do this in a method and not copy the code, for sake of avoiding mistakes